packages/codegen-doc/src/object.ts

```diff
@@ -64,3 +66,27 @@ export function isOperationRoot(
 ): boolean {
   return object ? Object.values(context.operationMap).includes(getObjectName(object)) : false;
 }
+
+/**
+ * Determine whether the object is valid to be output
+ *
+ * Use config skipComments to skip fields with comments containing specific strings
+ */
+export function isValidObject(context: PluginContext, object?: ObjectTypeDefinitionNode): boolean {
+  const skipComment = context.config.skipComments?.some(comment => object?.description?.value.includes(comment));
+
+  const connection = isConnection(object);
+  if (connection && object) {
+    // Check the type of this connection is valid
+    const rootTypeName = getObjectName(object).replace("Connection", "");
```

Comment:
Minor: this will cause a problem if we have a Connection entity in our system. I guess we wouldn't do that (it's a pretty similar word to integration, which is an entity name), but if we had ConnectionConnection then we'd do the wrong thing here 🙂